docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -151,6 +151,10 @@ described in [Teleport Upcoming Releases](../../upcoming-releases.mdx).
 - Minor updates and patches typically occur between one day and a week after a release.
 - Teleport Auth Service and Teleport Proxy Service updates are pushed to Teleport cloud accounts
   during your scheduled maintenance window.
+- Teleport Auth and Proxy service will not be automatically upgraded to the next major version with
+  Teleport agents on older major versions. An example would be the Teleport Auth and Proxy service are on
```

Comment:
"Auth and Proxy service" should be "Auth Service and Proxy Service" (here and below)